crates/solvers/src/tests/zeroex/market_order.rs

```diff
@@ -88,7 +88,7 @@ async fn sell() {
     }])
     .await;
 
-    let engine = tests::SolverEngine::new("zeroex", zeroex::config(&api.address)).await;
+    let engine = tests::SolverEngine::new("zeroEx", zeroex::config(&api.address)).await;
```

Comment:
nit: we don't really need to change this, do we?

Reply:
Reverted